app/models/condition.rb

```diff
@@ -145,7 +145,7 @@ def self.collect_children(ref, methods)
     result
   end
 
-  def self._subst_find(rec, inputs, expr)
+  def self._subst_find(rec, expr)
```

Comment:
@yrudman Are there any other callers of _subst_find or _subst that need to change due to the change in method signature?